client/src/components/Visualizations/PluginList.vue

```diff
@@ -137,7 +136,8 @@ export default {
             if (plugin.target == "_top") {
                 window.location.href = href;
             } else {
-                $("#galaxy_main").attr("src", href);
+                var galaxy_main_el = document.getElementById("galaxy_main");
+                galaxy_main_el.setAttribute("src", href);
```

Comment:
Thanks for working on this!

One minor nit that linting will complain about, and a tweak to use camelCase in js:

```suggestion
                const galaxyMainElement = document.getElementById("galaxy_main");
                galaxyMainElement.setAttribute("src", href);
```